src/wallet/wallet.h

```diff
@@ -1234,6 +1238,13 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     void NotifyTransactionLock(const CTransaction &tx, const llmq::CInstantSendLock& islock) override;
     void NotifyChainLock(const CBlockIndex* pindexChainLock, const llmq::CChainLockSig& clsig) override;
 
+    /** Load a CGovernanceObject into m_gobjects. */
+    bool LoadGovernanceObject(const CGovernanceObject& obj);
+    /** Store a CGovernanceObject in the wallet database. */
+    bool WriteGovernanceObject(const CGovernanceObject& obj);
+    /** Returns a vector containing pointers to the governance objects in m_gobjects */
+    std::vector<const CGovernanceObject*> GetGovernanceObjects();
```

Comment:
This doesn't apply to ALL governance objects correct? It only handles governance objects that your wallet cares about (because you made it)

Reply:
Correct

Reply:
We should probably adjust these comments then to make it a bit clearer